MasterServer/Program.cs

```diff
@@ -6,114 +6,146 @@
 
 namespace MasterServer
 {
-	class Program
+	internal static class Program
 	{
+		private static bool s_Daemon = false;
```

Comment:
I'd personally remove the prefixes used here. None of the Forge source uses that convention so let's not introduce that here. Please follow the https://github.com/BeardedManStudios/ForgeNetworkingRemastered/blob/master/CONTRIBUTING.md

Reply:
Done